examples/k8s-overlay-ingress/network-api-fix.yaml

```diff
@@ -0,0 +1,75 @@
+#! START OF COMMENT SECTION
+#! ------------------------
+#! We're overlaying the apiVersion used by falco charts
+#! We need to do so because we're rolling at least
+#! v1.22+ of Kubernetes ( as of 211230 )
+#! And this > https://github.com/falcosecurity/charts/blob/d2c6b91ff7339a5108abe23208212eb53b535463/falcosidekick/templates/_helpers.tpl#L38
+#! does not seem to work with the version value that the K3s distribution
+#! uses. For more background see > https://github.com/falcosecurity/charts/issues/288
+#! ----------------------
+#! END OF COMMENT SECTION
```

Comment:
This is helpful context that I think would belong in a longer description of the problem in the readme. 

> does not seem to work with the version value that the K3s distribution uses.

From what I can gather there is not any specifics on the K3 distribution in the issues linked, can you explain what makes this not work?

Reply:
I tried to do so in the `ReadMe.md` file.